packages/apollo-link-ddp/package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "apollo-link-ddp",
-  "version": "2.0.0",
+  "version": "2.1.0",
   "description": "Apollo Link using DDP",
   "main": "index.js",
   "peerDependencies": {
-    "apollo-link": "^1.0.0"
+    "apollo-link": "^1.0.0",
+    "apollo-utilities": "^1.1.0"
```

Comment:
This should be added to the README, where currently `graphql` is mentioned.

Reply:
Oh I'm sorry, I missed that!